CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+An action that has been ratified is said to be **enacted** once it has been activated on the network.
+Regardless of whether they have been ratified, actions may, however, be **dropped** without being **enacted** if, for example, a motion of no confidence is enacted.
+
+
+| Action                                         | Description                                                                                        |
+| ---                                            | ---                                                                                                |
+| 1. Motion of no-confidence                     | A motion to create a _state of no-confidence_ in the current Constitutional Committee  |
+| 2. New Constitutional Committee and/or quorum size | Changes to the members of the Constitutional Committee and/or to its signature threshold   |
+| 3. Updates to the Constitution                 | A modification to the off-chain Constitution, recorded as an on-chain hash of the text document                          |
+| 4. Hard-Fork[^2] Initiation                    | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |
+| 5. Protocol Parameter Changes                  | Any change to one or more updatable protocol parameters, excluding changes to major protocol versions ("hard forks")                                  |
+| 6. Treasury Withdrawals                        | Movements from the treasury, sub-categorized into small, medium or large withdrawals (based on the amount of Lovelace to be withdrawn). The thresholds for treasury withdrawals are discussed below.             |
+
+Any Ada holder can submit a governance action to the chain. They must provide a deposit of `Gov-Deposit` Lovelace, which will be returned when the action is finalized (whether it is **ratified**, has been **dropped**, or has **expired**).
+
+Note that a motion of no-confidence is an extreme measure that enables Ada holders to revoke the power that has been granted to the current Constitutional Committee. Any outstanding governance actions, including ones that the committee has ratified, will be dropped if the motion is enacted. As for other governance actions, votes ensue, followed by ratification or expiry.
+
+#### Ratification
+
+Governance actions are **ratified** through on-chain voting actions.  Different kinds of governance actions have different ratification requirements: depending on the type of governance action, an action will become ratified if some specific combination of the following occurs:
+
+* the Constitutional Committee approves of the action (`Quorum-Many` members vote 'yes');
+* the DReps approve of the action (the stake controlled by the DReps who vote 'yes' meets a certain threshold over those who vote 'no');
```

Comment:
This is a little surprising to me: I don't recall seeing a voting system that relied on a `yes - no` threshold rather than just a `yes` threshold. My guess is that this is related to the AVST thresholds:
- You want a certain level of participation for legitimacy, hence the AVST threshold
- If abstension is equivalent to voting no, then there is no incentive for people to vote no
- Therefore voting no must make the proposal materially more likely to fail than abstaining, hence the choice of the threshold.

But I would really like to see a rationale section explaining this if so.

Reply:
Apologizes, that is worded incorrectly. Or least it was not my understanding either, and I missed this change during the edits prior to releasing it. I am fairly confident that it should state that it is just a certain percentage of 'yes' votes to the total registered votes.

I will definitely write more in the rationale, as you suggested, thank you. The 'no' votes exist entirely for the on-chain audit trail, so that DReps can prove that they did not just forget to vote. The semantics regarding 'abstain' are, as far as I am concerned, still TBD. The natural way to handle them would be to remove the vote from the voting pool (ie subtracting from the denominator). This may or may not be a good idea, since given our current system this might lead to having the SPOs vote instead. I'm am definitely still working with the researchers on this question, and would also love the community to chime in.

Reply:
Can we be more precise, what is meant with "a certain threshold". Depending on this, the voting power devoted to NO votes can lead to a controlling situation.